### ISSUE TITLE
strong degradation tagger pipeline

### ISSUE THREAD

**Issue by thank267** [explosion/spaCy#2399]
I'm trying to train a Russian model with CLI train with morphology. But at 5-6 iterations there is a strong degradation tagger pipeline. The accuracy drops to ~ 25%. I use the UD Russian corpus. Without morphology, the accuracy does not decrease. Are there any considerations?

## Your Environment

* **spaCy version:** 2.0.11
* **Platform:** Linux-4.9.0-6-amd64-x86_64-with-debian-9.4
* **Python version:** 3.5.3
* **Models:** ru

**Comment by syllog1sm** [comment]
I'd love to have better health checks during training to diagnose these problems. For now the best advice I have isn't very satisfying: try reducing the learning rate, and perhaps also the batch size.

Essentially the neural network is getting "stuck" with weights from which SGD cant improve easily. Often this means some weights have a very large magnitude, or possibly NaN values.

There could also be a bug that means the objective is not consistent on your data. This happens more often for the parser though - -  it's less often a problem for the tagger.

**Comment by thank267** [comment]
I tried to increase token_vector_width and embed_size. But then the accuracy did not fall. Started with 26 and grew only to 46% ... Which is also not good:(

tag_map also checked for relevance

**Comment by thank267** [comment]
Intuitively, I also think the problem is in the weights. tag_map contains 730 labels.

**Comment by thank267** [comment]
@syllog1sm perfect! I decreased the learning rate and greatly increased batch size.
